plugins/setoid/setoid.ml

```diff
@@ -0,0 +1,24 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
```

Comment:
Do you want to refer to Rocq rather than Coq?  Also in the next two files.

Reply:
Indeed, an oversight due to the rebase of an old branch. Let me fix this.